app/_hub/moesif/kong-plugin-moesif/index.md

```diff
@@ -116,18 +132,34 @@ params:
       required: false
       default: "`{}`"
       description: An array of response body fields to mask.
+    - name: batch_size
+      required: false
+      default: "`200`"
+      description: Maximum batch size when sending to Moesif.
+    - name: event_queue_size
+      required: false
+      default: "`5000`"
+      description: Maximum number of events to hold in queue before sending to Moesif. In case of network issues when not able to connect/send event to Moesif, skips adding new to event to queue to prevent memory overflow.
```

Comment:
```suggestion
      description: Maximum number of events to hold in the queue before sending to Moesif. In case of network issues where the plugin is unable to connect or send an event to Moesif, skips adding new events to the queue to prevent memory overflow.
```